specs/electra/beacon-chain.md

```diff
@@ -70,6 +162,17 @@ The following values are (non-configurable) constants used throughout the specif
 | `MAX_EXECUTION_LAYER_EXITS` | `2**4` (= 16) |  *[New in Electra:EIP7002]* |
 | `MAX_ATTESTER_SLASHINGS_ELECTRA`   | `2**0` (= 1) | *[New in Electra:EIP7549]* |
 | `MAX_ATTESTATIONS_ELECTRA` | `2**3` (= 8) | *[New in Electra:EIP7549]* |
+| `MAX_WITHDRAWAL_REQUESTS_PER_PAYLOAD` | `uint64(16)` |
+| `MAX_PARTIAL_WITHDRAWALS_PER_PAYLOAD` | `uint64(2**3)` (= 8) | Maximum amount of partial withdrawals allowed in each payload |
```

Comment:
As a English-as-second-language person, I do not understand the reason to use `MAX_WITHDRAW_REQUESTS_PER_PAYLOAD` instead of `MAX_WITHDRAWAL_REQUESTS_PER_PAYLOAD` and `withdraw_requests` instead of `withdrawal_requests`. Could someone help me understand?

The only thing I know is that I have already written `withdrawal` instead of `withdraw` at least 50 times just today... xD

Reply:
yeah, this can be withdrawal